scalableminds/webknossos
# Volume Upload: skip empty buckets

### URL of deployed dev instance (used for testing):
- https://volumeuploadignoreemptybuckets.webknossos.xyz

### Steps to test:
- upload a volume tracing zip
- should still work
- upload a volume tracing zip with some all-zero bucket files (such as the one attached in [this post](https://discuss.webknossos.org/t/wkw-format-of-volume-tracings/1234/13))
- should also work (the above zip caused timeouts before this change, now took ~20s for me)
- create volume tracing, trace some, save, then erase data (brush with right mouse button), save. refresh. the now-zero data should be saved (not ignored)

### Issues:
- fixes #3548

------
